packages/react-core/src/components/Slider/examples/Slider.md

```diff
@@ -51,8 +65,16 @@ class DiscreteInput extends React.Component {
     const displayValue = step ? step.label : 0;
```

Comment:
Line above need `this.state.value` changed to `this.state.value1`:
`const step = this.steps.find(step => step.value === this.state.value1);`